apps/storybook/src/stories/datacom-loader.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect } from 'react';
+import { StoryFn } from '@storybook/react';
+import { DatacomLoader, DatacomButton } from '@datacom/endeavour-react';
+import styled from '@emotion/styled';
+
+type LoaderProps = React.ComponentProps<typeof DatacomLoader>;
+
+export default {
+  title: 'Loader',
+  component: DatacomLoader,
+  argTypes: {
+    label: {
+      name: 'Label',
+      defaultValue: 'Loading message',
+      description: 'Label for loading status.',
+      type: { name: 'string' },
+    },
+    loadingStatus: {
+      name: 'Loading Status',
+      defaultValue: 'default',
+      control: 'select',
+      description: 'Loading status is default if it is not set',
+      options: ['default', 'none', 'error', 'success'],
+    },
+  },
+};
+
+const Template: StoryFn<LoaderProps> = (args) => {
+  return <DatacomLoader></DatacomLoader>;
+};
+
+export const InlineLoader = (args) => {
+  const { label } = args;
+  return <DatacomLoader {...args}>{label}</DatacomLoader>;
+};
+
+export const WithLoadingStatus = () => {
+  const [loadingStatus, setLoadingStatus] = useState('none');
```

Comment:
Seeing error linting on line 68 complaining about your loadingStatus state not having the correct type. Maybe you can set the type on the useState hook to utilize the `LoadingStatusType`?